Crash report 2.8 - NPE in PostUploadService

```
java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:299)
       at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:273)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:124)
       at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:307)
       at java.util.concurrent.FutureTask.run(FutureTask.java:137)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:230)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
       at java.lang.Thread.run(Thread.java:856)
Caused by: java.lang.NullPointerException
       at org.wordpress.android.util.PostUploadService$UploadPostTask.doInBackground(PostUploadService.java:271)
       at org.wordpress.android.util.PostUploadService$UploadPostTask.doInBackground(PostUploadService.java:131)
       at android.os.AsyncTask$2.call(AsyncTask.java:287)
       at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
       at java.util.concurrent.FutureTask.run(FutureTask.java:137)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:230)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
       at java.lang.Thread.run(Thread.java:856)
```

--- a/src/org/wordpress/android/util/PostUploadService.java
+++ b/src/org/wordpress/android/util/PostUploadService.java
@@ -265,14 +265,12 @@ public class PostUploadService extends Service {
                         String imgPath = m.group(1);
                         if (!imgPath.equals("")) {
                             MediaFile mf = WordPress.wpDB.getMediaFile(imgPath, post);
-
-                            if (mf.isVideo()) {
-                                hasVideo = true;
-                            } else {
-                                hasImage = true;
-                            }
-
                             if (mf != null) {
+                                if (mf.isVideo()) {
+                                    hasVideo = true;
+                                } else {
+                                    hasImage = true;
+                                }
                                 String imgHTML = uploadMediaFile(mf, blog);
                                 if (imgHTML != null) {
                                     if (x == 0) {

